pipeline/backend/backend.go

```diff
@@ -29,11 +29,11 @@ var (
 	backends       []types.Backend
 )
 
-func Init(ctx context.Context) {
+func Init() {
```

Comment:
can we move this change into it's own pull and merge it first?

Reply:
I've reverted this change. Not even sure why it ended in this PR, as at least the linter doesn't complain.